pkgs/development/python-modules/configargparse/default.nix

```diff
@@ -0,0 +1,18 @@
+{ stdenv, lib, buildPythonPackage, fetchurl, pythonPackages }:
+
+buildPythonPackage rec {
+  name = "ConfigArgParse-${version}";
```

Comment:
`pname = "ConfigArgParse";`
and drop the name attribute.
This is something new in `buildPythonPackage`.